src/igtcloud/client/tools/common.py

```diff
@@ -47,11 +51,13 @@ def find_institute_by_name(project_id: str, institute_name: str) -> Optional[Ins
 def find_project_and_institutes(project_name: str, institute_name: str = None) -> Tuple[Optional[Project], List[Institute]]:
     institutes = list()
     project = find_project_by_name(project_name)
-    if not project:
-        return None, institutes
 
     if institute_name:
-        institute = find_institute_by_name(project.id, institute_name)
+        if project:
+            institute = find_institute_by_name(institute_name, project_id=project.id)
+        else:
+            institute = find_institute_by_name(institute_name)
+
         if institute:
             institutes.append(institute)
     else:
```

Comment:
In the else clause, you should check if project is None, if so return (None, institutes)